runtime/j9vm/java11vmi.c

```diff
@@ -603,6 +606,7 @@ exportPackageToModule(J9VMThread * currentThread, J9Module * fromModule, const c
 	J9Package * const j9package = getPackageDefinition(currentThread, fromModule, package, &retval);
 	if (NULL != j9package) {
 		if (isModuleDefined(currentThread, toModule)) {
+			Trc_MODULE_invokeHashTableAtPut(currentThread, "exportPackageToModule(exportsHashTable)", j9package->exportsHashTable, &toModule, toModule, "false");
```

Comment:
I'm less sure about what to log here.... though by analogy to the above, it likely should be the `j9package`

Reply:
Will log `j9package` here.

This issue investigation didn't involve much about `j9package` or `HashTable` itself, it was mainly to correlate the `Java Module object` with native `J9Module` structure underneath.

The core dump file can indicate the mismatch and identify the bad object however it was difficult to crawl back when the bad object was created and how it becomes invalid. When the failure is intermittent and takes long hours to reproduce, trace points are the friends to record what happened. 

So I would like to put both `j9package` and `j9package->exportsHashTable` into the log since it doesn't require much extra cost.